packages/expo-camera/ios/ExpoCameraUtils.swift

```diff
@@ -0,0 +1,175 @@
+import AVFoundation
+import CoreMotion
+
+struct ExpoCameraUtils {
+  static func device(with mediaType: AVMediaType, preferring position: AVCaptureDevice.Position) -> AVCaptureDevice? {
+    return AVCaptureDevice.default(.builtInWideAngleCamera, for: mediaType, position: position)
+  }
+
+  static func deviceOrientation(
+    for accelerometerData: CMAccelerometerData,
+    default orientation: UIDeviceOrientation
+  ) -> UIDeviceOrientation {
+    if accelerometerData.acceleration.x >= 0.75 {
+      return .landscapeRight
+    }
+    if accelerometerData.acceleration.x <= -0.75 {
+      return .landscapeLeft
+    }
+    if accelerometerData.acceleration.y <= -0.75 {
+      return .portrait
+    }
+    if accelerometerData.acceleration.y >= 0.75 {
+      return .portraitUpsideDown
+    }
+
+    return orientation
+}
+
+  static func videoOrientation(for interfaceOrientaion: UIInterfaceOrientation) -> AVCaptureVideoOrientation {
+    switch interfaceOrientaion {
+    case .portrait:
+      return .portrait
+    case .landscapeLeft:
+      return .landscapeLeft
+    case .landscapeRight:
+      return .landscapeRight
+    case .portraitUpsideDown:
+      return .portraitUpsideDown
+    default:
+      return .portrait
+    }
+  }
+
+  static func videoOrientation(for deviceOrientaion: UIDeviceOrientation) -> AVCaptureVideoOrientation {
+    switch deviceOrientaion {
+    case .portrait:
+      return .portrait
+    case .portraitUpsideDown:
+      return .portraitUpsideDown
+    case .landscapeLeft:
+      return .landscapeLeft
+    case .landscapeRight:
+      return .landscapeRight
```

Comment:
I'm not sure, but is it inverted?
I think the original implementation was like the below.
https://github.com/expo/expo/pull/22604/files#diff-d398be0ed3360cdf239f9f8b76675856ebc7bfe4425c789029678c0d344c1022
```mm
+ (AVCaptureVideoOrientation)videoOrientationForDeviceOrientation:(UIDeviceOrientation)orientation
{
  switch (orientation) {
    case UIDeviceOrientationPortrait:
      return AVCaptureVideoOrientationPortrait;
    case UIDeviceOrientationPortraitUpsideDown:
      return AVCaptureVideoOrientationPortraitUpsideDown;
    case UIDeviceOrientationLandscapeLeft:
      return AVCaptureVideoOrientationLandscapeRight;
    case UIDeviceOrientationLandscapeRight:
      return AVCaptureVideoOrientationLandscapeLeft;
    default:
      return AVCaptureVideoOrientationPortrait;
  }
}
```